modules/govuk/manifests/deploy.pp

```diff
@@ -11,19 +10,17 @@
   include govuk_python
   include unicornherder
 
-  validate_bool($setup_actionmailer_ses_config)
   validate_bool($actionmailer_enable_delivery)
 
   anchor { 'govuk::deploy::begin': }
 
   # These resources are required, but should not refresh apps.
   class { 'govuk::deploy::setup':
-    actionmailer_enable_delivery  => $actionmailer_enable_delivery,
-    setup_actionmailer_ses_config => $setup_actionmailer_ses_config,
-    aws_ses_smtp_host             => $aws_ses_smtp_host,
-    aws_ses_smtp_username         => $aws_ses_smtp_username,
-    aws_ses_smtp_password         => $aws_ses_smtp_password,
-    require                       => Anchor['govuk::deploy::begin'],
+    actionmailer_enable_delivery => $actionmailer_enable_delivery,
+    aws_ses_smtp_host            => $aws_ses_smtp_host,
+    aws_ses_smtp_username        => $aws_ses_smtp_username,
+    aws_ses_smtp_password        => $aws_ses_smtp_password,
```

Comment:
Can we remove these too, as I think they were only used to generate the ActionMailer configuration file?